megamek/src/megamek/common/MechView.java

```diff
@@ -44,6 +45,11 @@
  * @since January 20, 2003
  */
 public class MechView {
+    public static enum ViewFormatting {
```

Comment:
This aren't big deals, but I'd encourage everyone to give classes their own file. Our existing classes are all too long. Also, as constants, the enum values are typically written in capitals (DISCORD...)